src/lib/cli/format.ts

```diff
@@ -254,3 +254,43 @@ export const formatBytes = (
 export const formatMetricBytes = ( bytes: number, decimals = 2 ): string => {
 	return formatBytes( bytes, decimals, 1000 );
 };
+
+/*
+ * Get the duration between two dates
+ *
+ * @param {Date} from The start date
+ * @param {Date} to  The end date
+ * @returns {string} The duration between the two dates
+ */
+export function formatDuration( from: Date, to: Date ): string {
+	const millisecondsPerSecond = 1000;
+	const millisecondsPerMinute = 60 * millisecondsPerSecond;
+	const millisecondsPerHour = 60 * millisecondsPerMinute;
+	const millisecondsPerDay = 24 * millisecondsPerHour;
+
+	const duration = to.getTime() - from.getTime();
+	if ( duration < 0 ) return '0 second';
```

Comment:
```suggestion
	if ( duration < 1000 ) return '0 second';
```

Edge case: If there's 999 milliseconds remaining and if I calculated correctly, it'd mean that we have 0 seconds, 0 days, 0 hours, 0 minutes, and we'd be returning a blank string!

I think we should fix this - 1 second is a long time 😅

Reply:
Sure. Thanks for being diligent!! 🎉